backend/mora/service/manager.py

```diff
@@ -13,20 +13,93 @@
 
 """
 import uuid
+import flask
 
 from . import address
 from . import handlers
+from . import orgunit
+from . import facet
+from . import employee
 from .. import common
 from .. import lora
 from .. import mapping
 from .. import util
 from .. import validator
 
+SEARCH_FIELDS = {
+    'e': 'tilknyttedebrugere',
+    'ou': 'tilknyttedeenheder'
+}
 
-class ManagerRequestHandler(handlers.OrgFunkRequestHandler):
+
+class ManagerRequestHandler(
+    handlers.OrgFunkRequestHandler,
+    handlers.OrgFunkReadingRequestHandler
+):
+
+    __slots__ = ()
     role_type = 'manager'
     function_key = mapping.MANAGER_KEY
 
+    @classmethod
+    def get_one_mo_object(cls, effect, start, end, funcid):
+        c = common.get_connector()
+
+        person = mapping.USER_FIELD.get_uuid(effect)
+        manager_type = mapping.ORG_FUNK_TYPE_FIELD.get_uuid(effect)
+        manager_level = mapping.MANAGER_LEVEL_FIELD.get_uuid(effect)
+        addresses = list(mapping.FUNCTION_ADDRESS_FIELD.get_uuids(effect))
+        responsibilities = list(mapping.RESPONSIBILITY_FIELD.get_uuids(effect))
+        org_unit = mapping.ASSOCIATED_ORG_UNIT_FIELD.get_uuid(effect)
+
+        func = {
+            mapping.UUID: funcid,
+            mapping.RESPONSIBILITY: sorted([
+                facet.get_one_class(c, classid, classobj)
+                for classid, classobj in c.klasse.get_all(
+                    uuid=responsibilities
+                )
+            ], key=lambda r: r["name"]),
```

Comment:
```suggestion
            ], key=lambda r: r[mapping.NAME]),
```

(Or use `operator.itemgetter()` if you prefer.)